2025-09-09 01:25:08.60
9eb594bd
User
raise rewrite_traceback_stack(source=source)
2025-09-09 01:25:08.60
9eb594bd
User
File "/home/runner/workspace/templates/admin/pending_duties.html", line 1, in top-level template code
2025-09-09 01:25:08.60
9eb594bd
User
{% extends "base.html" %}
2025-09-09 01:25:08.60
9eb594bd
User
File "/home/runner/workspace/templates/base.html", line 295, in top-level template code
2025-09-09 01:25:08.60
9eb594bd
User
{% block content %}{% endblock %}
2025-09-09 01:25:08.60
9eb594bd
User
^^^^^^^^^^^^^^^^^^^^^^^^^
2025-09-09 01:25:08.60
9eb594bd
User
File "/home/runner/workspace/templates/admin/pending_duties.html", line 254, in block 'content'
2025-09-09 01:25:08.60
9eb594bd
User
2025-09-09 01:25:08.60
9eb594bd
User
^^^^^^^^^^^^^^^^^^^^^^^^^
2025-09-09 01:25:08.60
9eb594bd
User
File "/home/runner/workspace/.pythonlibs/lib/python3.11/site-packages/flask/app.py", line 1121, in url_for
2025-09-09 01:25:08.60
9eb594bd
User
return self.handle_url_build_error(error, endpoint, values)
2025-09-09 01:25:08.60
9eb594bd
User
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-09-09 01:25:08.60
9eb594bd
User
File "/home/runner/workspace/.pythonlibs/lib/python3.11/site-packages/flask/app.py", line 1110, in url_for
2025-09-09 01:25:08.60
9eb594bd
User
rv = url_adapter.build( # type: ignore[union-attr]
2025-09-09 01:25:08.60
9eb594bd
User
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-09-09 01:25:08.60
9eb594bd
User
File "/home/runner/workspace/.pythonlibs/lib/python3.11/site-packages/werkzeug/routing/map.py", line 924, in build
2025-09-09 01:25:08.60
9eb594bd
User
raise BuildError(endpoint, values, method, self)
2025-09-09 01:25:08.60
9eb594bd
User
werkzeug.routing.exceptions.BuildError: Could not build url for endpoint 'serve_file' with values ['filename']. Did you mean 'driver.profile' instead?
2025-09-09 02:09:19.63
9eb594bd
User
[2025-09-08 20:39:19 +0000] [25] [INFO] Handling signal: term
2025-09-09 02:09:19.63
9eb594bd
User
[2025-09-08 20:39:19 +0000] [30] [INFO] Worker exiting (pid: 30)
2025-09-09 02:09:20.14
9eb594bd
User
[2025-09-08 20:39:20 +0000] [25] [INFO] Shutting down: Master
2025-09-09 02:12:26.82
9eb594bd
User
[2025-09-08 20:42:26 +0000] [26] [INFO] Starting gunicorn 23.0.0
2025-09-09 02:12:26.82
9eb594bd
User
[2025-09-08 20:42:26 +0000] [26] [INFO] Listening at: http://0.0.0.0:5000 (26)
2025-09-09 02:12:26.82
9eb594bd
User
[2025-09-08 20:42:26 +0000] [26] [INFO] Using worker: sync
2025-09-09 02:12:26.83
9eb594bd
User
[2025-09-08 20:42:26 +0000] [31] [INFO] Booting worker with pid: 31
2025-09-09 02:12:37.07
9eb594bd
User
DEBUG:google.cloud.storage._opentelemetry_tracing:This service is instrumented using OpenTelemetry. OpenTelemetry or one of its components could not be imported; please add compatible versions of opentelemetry-api and opentelemetry-instrumentation packages in order to get Storage Tracing data.